messaging/src/test/java/org/axonframework/commandhandling/distributed/ConsistentHashTest.java

```diff
@@ -109,4 +110,14 @@ void testConflictingHashesDoNotImpactMembership() {
         assertEquals(member1.name(), consistentHash.getMembers().iterator().next().name());
         assertEquals(consistentHash.getMembers(), consistentHashModified.getMembers());
     }
+
+    @Test
+    void testEquals1() throws Exception {
+        ConsistentHash consistentHash = new ConsistentHash((s) -> "fixed").with(member1, 1, AcceptAll.INSTANCE);
+        ConsistentHash consistentHashModified = consistentHash
```

Comment:
The `ConsistentHash` instance on line 116 is, technically speaking, already modified.
Because of the `with` invocation to be exact.
Hence, the second and third modifications aren't necessary (although I think they're copied from line 106/107 (which is fine by the way!)).

I wouldn't expect these adjustments to the test subject to matter for the test.
As such, I'd wager that a simple `ConsistentHash` instance, with either (1) an initial state (e.g. `(s) -> "fixed"`) or (2) a default instance with modification is sufficient for your test case.

Reply:
Awesome, thanks for that, @lacinoire!